internal/verify/verify.go

```diff
@@ -36,13 +34,24 @@ type SuiteResult struct {
 	Tests   []TestResult `json:"tests"`
 }
 
+type TestName struct {
+	TableTestName string `json:"name"`
+	PrincipalKey  string `json:"principal"`
+}
+
+func (r TestName) String() string {
+	return fmt.Sprintf("'%s' by principal '%s'", r.TableTestName, r.PrincipalKey)
+}
+
 type TestResult struct {
-	Name    string `json:"name"`
-	Skipped bool   `json:"skipped,omitempty"`
-	Failed  bool   `json:"failed,omitempty"`
-	Error   string `json:"error,omitempty"`
+	Name    *TestName `json:"case"`
```

Comment:
nit: Any reason why this has to be a pointer?

Reply:
To exclude TestName struct from JSON output if the field is empty. But `Name` is always set, so there's no such problem. I can remove the pointer.